packages/Utils/src/DTK_ConfigDefs.hpp

```diff
@@ -15,7 +15,7 @@
 #ifndef DTK_CONFIGDEFS_HPP
 #define DTK_CONFIGDEFS_HPP
 
-#include "DataTransferKitUtils_config.hpp"
+#include "DataTransferKit_config.hpp"
```

Comment:
OK so now we have a single config file?

Reply:
Correct.  I plan need on revisiting other subpackages later and do the same.  It is trickier for them because it is tied to our ETI.